v5/core/cr_authenticator.go

```diff
@@ -430,44 +470,8 @@ func (authenticator *ComputeResourceAuthenticator) RequestToken() (*IamTokenServ
 	return tokenResponse, nil
 }
 
-// retrieveCRToken retrieves the CR token for the current compute resource.
```

Comment:
The diffs starting here to the end of the file are a bit confusing (at least to me!), but if you view the whole file, you can see that things are simplified quite a bit.